corehq/form_processor/submission_context.py

```diff
@@ -0,0 +1,43 @@
+import lxml.etree
+from memoized import memoized
+import xml2json
+
+from corehq.form_processor.extension_points import get_form_submission_context_class
+from corehq.form_processor.utils.metadata import scrub_meta
+
+from couchforms.util import legacy_notification_assert
+
+
+class SubmissionFormContext(object):
+    def __init__(self, submission_post):
+        self.submission_post = submission_post
+        self._instance = submission_post.instance
+
+    def __enter__(self):
+        return self
+
+    def __exit__(self, *exc_info):
+        pass
```

Comment:
Why is this a context manager? It looked like `ICDSSubmissionFormContext` does not override these.

Reply:
Yea, there's no real need. Originally when I suggested it I had some other thought in mind.

Reply:
updated https://github.com/dimagi/commcare-hq/pull/28409/commits/8df89511a47d808a545a0766d363e3a498b2038f
corresponding icds change: https://github.com/dimagi/commcare-icds/pull/41/commits/11fb8e05ad51246610c60f7bdc4fa39c56f0d51b